docs/tutorials/projects/modernize-retro-robot.md

```diff
@@ -3,10 +3,608 @@ title: "Modernize a 1980s robot"
+        "width_mm": 220
+      },
+      "depends_on": [
+        "base-l",
+        "base-r"
+      ]
+    }
+  ]
+}
+```
+
+Click **Save config** in the bottom left corner of the screen.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Test the base and motor components
+
+MAIV is still in pieces, but its time to ensure that the motors and base controls are working.
+Check to see if you might have something that you can use to set MAIV's base upon so the motorized wheels can spin freely.
+Or, if the wires between the base and the torso are long enough to allow small movements of the base you can leave the base on the flat surface.
+
+Navigate to your robot's [Control](/manage/fleet/robots/#control) tab.
+Click on the base panel and use the arrows to control the base.
+Ensure the motors on both sides are working, and that the wheels are spinning as expected.
+If the wheels on a given side are spinning in the opposite direction of what you would expect, go back to the **Config** tab, toggle `Direction Flip` for the appropriate motor, **Save config** and repeat the test steps.
+
+{{<video webm_src="../../img/maiv/maiv_base_test.webm" mp4_src="../../img/maiv/maiv_base_test.mp4" alt="MAIV base test movements" max-width="300px">}}
+
+From the **Control** tab you can also test the individual motors directly.
+First select the desired direction and power percent, start the motor by clicking `RUN` and stop the motor by clicking `STOP`.
+
+### MAIV's head
+
+In order to give MAIV a bit more personality and capability, let's wire MAIV's neck motor and eyes.
+We'll also add a camera so MAIV can see the world.
+
+#### Convert MAIV's eyes to LEDs
+
+Because MAIV's torso is already open, the neck and head is detached.
+Turn the head over, locate and unscrew the two screws holding MAIV's face plate.
+You will be replacing the current incandescent bulbs with more energy-efficient LEDs that can be controlled directly from your Raspberry Pi.
+In order to access the eye wiring, locate the screws on the underside of the head and unscrew them.
+
+<div class="td-max-width-on-larger-screens">
+<img src="../../img/maiv/maiv_eyes.jpg" alt="Omnibot MAIV eye plate." class="alignleft" title="Omnibot MAIV eye plate." style="max-width: 250px;" />
+</div>
+
+Remove the eye shield, unscrew the incandescent bulbs and screw in the LED bulbs.
+
+The original wires to the eye bulb sockets won't be long enough to reach the Raspberry Pi.
+Cut the original wires (leave some length) and strip the ends.
+You will control both eyes at once, so use a 3-terminal quick connect to connect both eyes positive wires and a third long breadboard wire.
+Repeat the same with another 3-terminal quick connect for the negative wires.
+
+Now, run the long breadboard wires to through the robot neck, torso, and into the base.
+
+With the robot powered off:
+
+* Plug the positive breadboard wire into pin 1 on the Raspberry Pi (3.3V).
+* Plug the negative breadboard wire into pin 18 (GPIO 24) on the Pi.
```

Comment:
just for my education: why isn't this going from GPIO to ground?

Reply:
Either would work - discussion here: https://forums.raspberrypi.com/viewtopic.php?t=234598

Reply:
thanks!